lib/Conversion/XTenToLinalg.cpp

```diff
@@ -113,37 +169,35 @@ class XTenBinaryOpConversion : public ConversionPattern {
                   ConversionPatternRewriter &rewriter) const override {
     auto loc = op->getLoc();
 
-    auto A = MemRefTypeCast(rewriter, operands[0]);
-    auto B = MemRefTypeCast(rewriter, operands[1]);
+    Value A = ToBuiltinTensorTypeCast(rewriter, operands[0]);
+    Value B = ToBuiltinTensorTypeCast(rewriter, operands[1]);
```

Comment:
@BraunPhilipp - here is one place where I replaced memref-based with tensor ops.